XML/TestCases/FunctionalTests-KernelDebug.xml

```diff
@@ -0,0 +1,13 @@
+<TestCases>
+    <test>
+        <testName>KERNEL-DEBUG-BASIC</testName>
+        <testScript>KERNEL-DEBUG-BASIC.ps1</testScript>
+        <files>.\Testscripts\Linux\check_traces.sh,.\Testscripts\Linux\utils.sh</files>
+        <setupType>OneVM</setupType>
+        <Platform>HyperV</Platform>
+        <Category>Functional</Category>
+        <Area>Kernel-Debug</Area>
+        <Tags>kernel-debug</Tags>
```

Comment:
kernel-debug -> boot

Reply:
Updated, looks that only this case belongs to boot tags, is it expected, yes?